src/rules/javascript/only-prefs-in-defaults.js

```diff
@@ -1,12 +1,15 @@
+import path from 'path';
+
 import { ONLY_PREFS_IN_DEFAULTS } from 'messages/javascript';
 import { getRootExpression } from 'utils';
 
+
 export default {
   create(context) {
-    var filename = context.getFilename();
+    var relPath = path.relative(process.cwd(), context.getFilename());
 
```

Comment:
This might need a bit more work and testing. I'd love to forward the actual package path to the rules eventually but for now this seems to work just fine. (also this rule will be removed anyway but it's a good example)